### ISSUE TITLE
Image Crop

### ISSUE THREAD

**Issue by amirakbari** [PHPOffice/PHPPresentation#398]
Is there any way to crop images on phppresentation ?

**Comment by juliohds** [comment]
you can crop before send

**Comment by amirakbari** [comment]
yes but Microsoft PowerPoint have multiple option for crop in slides that save original image and show a view of cropped image and user can change dimension of crop

**Comment by Progi1984** [comment]
@amirakbari Hi, Could you send me a sample file with a cropped image, please, for implementing the feature ?

**Comment by amirakbari** [comment]
@Progi1984 
Sure :) 
[croped-image.pptx](https://github.com/user-attachments/files/16845583/croped-image.pptx)
